src/test/java/org/zalando/problem/validation/ValidationProblemModuleTest.java

```diff
@@ -0,0 +1,48 @@
+package org.zalando.problem.validation;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonassert.JsonAssert;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.zalando.problem.ProblemModule;
+import org.zalando.problem.spring.web.advice.validation.ConstraintViolationProblem;
+import org.zalando.problem.spring.web.advice.validation.Violation;
+
+import javax.ws.rs.core.Response;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+@Slf4j
+public class ValidationProblemModuleTest {
+
+    @Test
+    public void serializeConstraintViolationUsingMixinAnnotations() throws JsonProcessingException {
+        ObjectMapper mapperWithDisabledAutoDetection = new ObjectMapper()
+                .configure(MapperFeature.AUTO_DETECT_FIELDS, false)
```

Comment:
You could use `disable(MapperFeature.AUTO_DETECT_FIELDS)` to make it a bit shorter.